app/views/users/_app_permissions.html.erb

```diff
@@ -0,0 +1,36 @@
+<table class="table table-bordered table-striped table-on-white">
+  <thead>
+    <tr class="table-header">
+      <th>Application</th>
+      <% unless user_object.api_user? %>
+          <th>Has access?</th>
+      <% end %>
+      <th><%= "Other" unless user_object.api_user? %> Permissions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @application_permissions.each do |application, permissions| %>
+      <% if permissions.present? %>
+        <tr>
+          <td>
+            <% if application.retired? %>
+                <del><%= application.name %></del>
+            <% else %>
+                <%= application.name %>
+            <% end %>
+          </td>
+          <% if user_object.api_user? %>
+            <td></td>
```

Comment:
We don't render the `Has access?` column header for API users so we dont' need to render an empty cell for it here.